drivers/serial/Kconfig

```diff
@@ -651,24 +651,9 @@ menuconfig PSEUDOTERM
 
 if PSEUDOTERM
 
-choice
-	prompt "PTY model"
-	default PSEUDOTERM_BSD if DISABLE_PSEUDOFS_OPERATIONS
-	default PSEUDOTERM_SUSV1 if !DISABLE_PSEUDOFS_OPERATIONS
-
-config PSEUDOTERM_BSD
-	bool "BSD style"
-	---help---
-		Deprecated BSD style PTYs.
-
-		Master: /dev/ptyN
-		Slave: /dev/ttypN
-
-		Where N is the minor number
-
 config PSEUDOTERM_SUSV1
```

Comment:
Do we need `PSEUDOTERM_SUSV1` option now? It was one of the choice values and now seems that there is no choice

Reply:
Here is the possible combination:

1. Enable PSEDOTERM only->BSD style only
2. Enable PSEDOTERM plus PSEDOTERM_SUSV1->BSD and SUSv1 style

Since PSEUDOTERM_SUSV1 need an additional ptmx driver, user can disable this to save code size if they want. On the other hand, the additional consumption of BSD style is very small(a couple of if/else), it's more simple to enable it automatically if user already turn on PSEDOTERM.

Reply:
I see. Thank you.